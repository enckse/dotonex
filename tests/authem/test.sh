_test() {
    {{ range $key, $vlan := .VLANs }}
        echo {{ $vlan }}
    {{ end }}
    {{ range $key, $sys := .Systems }}
        echo {{ $sys }}
    {{ end }}
    {{ range $key, $user := .Users }}
        echo {{ $user.UserName }}
        echo {{ $user.LoginName }}
        echo {{ $user.Password }}
        {{ range $skey, $sys := $user.Systems }}
            echo {{ $sys }}
        {{ end }}
    {{ end }}
}

_test > bin/script.stdout
