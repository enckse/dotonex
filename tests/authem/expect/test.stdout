{dev 1 development [] axyz test}
{prod 2 waejfaeojai [] axyz test}
{y test other xyz}
test
test
fpllngzieyoh43e0133ols6k1hh2gdny
{{y test other xyz} a [aabbccddeeff]}
test2
test2
fpllngzieyoh43e0133ols6k1hh2gdny
{{y test other xyz} a [aabbccddee11]}
test3
test3
fpllngzieyoh43e0133ols6k1hh2gdny
test4
test4
fpllngzieyoh43e0133ols6k1hh2gdny
{{y test other xyz} a [aa11ccddeeff]}
test5
test5
fpllngzieyoh43e0133ols6k1hh2gdny
test6
test6
fpllngzieyoh43e0133ols6k1hh2gdny
{{y test other xyz} a [aabbccddee12]}
